chart/values.yaml

```
@@ -1919,6 +1919,19 @@ dags:
 
     extraVolumeMounts: []
     env: []
+     # Change permissions on the checked-out files to the specified mode.
+     # - name: GIT_SYNC_PERMISSION
+     #   value: "0755"
+     # The time to wait before retrying a failed --exechook-command.
+     # - name: GIT_SYNC_EXECHOOK_BACKOFF
+     #   value: "3s"
+     # An optional command to be executed after syncing a new hash of the remote repository.
+     # - name: GIT_SYNC_EXECHOOK_COMMAND
+     #   value: "./scripts/entrypoint.sh"
+     # The timeout for the --exechook-command.
+     # - name: GIT_SYNC_EXECHOOK_TIMEOUT
+     #   value: "30s"
```

```suggestion
     # Supported env vars for gitsync can be found at https://github.com/kubernetes/git-sync
     # - name: ""
     #   value: ""
     
```

Let's be general here, I don't want to end up duplicating the whole gitsync docs here.

---

I was thinking of expressing only the frequently used env values, but it would be better to give a general guide.